app/schemas/de.tum.in.tumcampusapp.database.TcaDb/10.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 10,
```

Comment:
When you add tables, you have to increase the database version!

Reply:
Yes, otherwise it will crash for anyone trying it out. Yes, you should add migrations and increase the version number for each change in the DB.

Reply:
Version increment done in #944